doc/source/user_guide/visualization.rst

```diff
@@ -6,6 +6,11 @@
 Chart visualization
 *******************
 
+
+.. note::
+
+   The examples below assume that you're using `Jupyter <https://jupyter.org/>`_.
+
```

Comment:
suggestion, add: 
```
    If one is implementing a script or is executing the Python interpreter
    from a command line interface, one may opt for ``plt.savefig('$filename')``
    in lieu of the ``$object.plot()`` functions found in this tutorial.

    ``plt.savefig()`` saves the pandas object as an image and accepts a variety  
    of filetypes, including png, pdf, jpg, and svg. 
```

Reply:
This isn't quite right, there's a button to press to suggest changes, check the article I linked (step 6)

Reply:
IMO I don't think this note is entirely necessary. Since the rest of the documentation mentions using the matplotlib API, I think further matplotlib actions can be independently looked-up by the user.

Reply:
That's a good point - let's keep as-is then